src/helpers/dates.ts

```diff
@@ -136,6 +136,10 @@ export class DateTime {
   setSeconds(seconds: number) {
     return this.jsDate.setUTCSeconds(seconds);
   }
+
+  copy() {
+    return DateTime.fromTimestamp(this.getTime());
+  }
```

Comment:
I wonder if this is a good method. it looks like a new constructor, and has only 1 usage...
Also, it only copies the time.